proposal.md

```diff
@@ -461,22 +461,53 @@ their package evolves. Note that CRAN permits dependencies on Bioconductor
 packages, as well as other repositories specified in the 
 `Additional_repositories` field of the `DESCRIPTION` file.
 
-Reverse dependencies may be identified with `tools::dependsOnPkgs`. Dependency 
-graphs may be created using `tools::package_dependencies`,
-based on a package database like that returned by `utils::available.packages`. Reverse dependencies back to CRAN packages from Bioconductor packages may be found by updating the package database repository list from `utils::available.packages` before running `tools::package_dependencies`.
+Forward or reverse dependencies may be identified with `tools::package_dependencies`,
+based on a package database like that returned by `utils::available.packages`.
+Dependencies from either CRAN or Bioconductor can be found by 
+setting the `repos` argument of `utils::available.packages` to  `BiocManager::repositories()`. 
 
 `utils::update.packages` is useful for updating dependencies when trying out changes to a package. `utils::sessionInfo` is helpful in tracking changes caused by upstream updates.
 
 `tools::check_packages_in_dir` can be used to check the reverse dependencies of a package (or set of packages).
 
-CONSIDER: checked, `sessioninfo::session_info`, revdepcheck, prefixer, rcheology, tools like drat and miniCRAN
-
 WRE reference: [Package Dependencies](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-Dependencies).
 
-
-
-SEE ALSO:  
-<https://github.com/ropensci-archive/PackageDevelopment/blob/master/README-NOT.md#dependency-management>, https://github.com/IndrajeetPatil/awesome-r-pkgtools?tab=readme-ov-file#dependency-management-%EF%B8%8F
+ - `r pkg("attachment")` provides helpers to update forward dependencies in your 
+ DESCRIPTION as required by changes to `.R` or `.Rmd` files, and to quickly 
+ install missing packages declared in a DESCRIPTION file.
+ - `r pkg("rcheology")` provides a dataset of functions in all base and 
+ recommended packages of R from version 0.50. This, or its companion 
+ [rcheology Shiny app](https://hughjonesd.shinyapps.io/rcheology/) can help 
+ to determine the minimum version of R on which a package depends.
+ - `r pkg("backports")` provides reimplementations of functions introduced or 
+ changed since R v3.0.0. This enables package developers to maintain 
+ compatability with older versions of R when using newer functionality.
+ - `r pkg("sessioninfo")` provides `session_info()` as an alternative to 
+ `utils::sessionInfo`, which returns richer information on add-on packages. 
```

Comment:
I would also mention packages are alphabetically ordered on `session_indo()`. Or not if it is mentioned before (see previous comment)